pym/portage/dbapi/bintree.py

```diff
@@ -330,6 +330,7 @@ def __init__(self, _unused=DeprecationWarning, pkgdir=None,
 				"USE_EXPAND", "USE_EXPAND_HIDDEN", "USE_EXPAND_IMPLICIT",
 				"USE_EXPAND_UNPREFIXED"])
 			self._pkgindex_default_pkg_data = {
+				"BDEPEND" : "",
```

Comment:
Internal indentation is inconsistent with the following lines.

Reply:
But it's consistent with the shorter entries following them.